_sources/Unit9-Inheritance/topic-9-7-Object.rst

```diff
@@ -462,7 +552,7 @@ Work in pairs to write the following code and test each part before moving on to
          public void test1()
          {
              String output = getMethodOutput("main");
-             String expect = "Armani Smith, 1500.0\nDakota Jones, 1500.0, 4.5\ntrue";
+             String expect = "Armani Smith, 1500.0\nDakota Jones, 1500.0, 4.5";
```

Comment:
Is Armani Smith still printed out? I wasn't sure if you deleted that.

Reply:
It still is.